paz/abstract/loader.py

```diff
@@ -75,3 +75,81 @@ def num_classes(self):
             return len(self.class_names)
         else:
             raise ValueError('class names are not a list')
+
+
+class Generator(object):
```

Comment:
I think we should move this outside along with the data generator for VVAD. I feel that we can make it better but it might take some time. What do you think?